composer.json

```diff
@@ -49,6 +49,10 @@
     "conflict": {
        "thecodingmachine/safe": "<1.1.3"
     },
+    "scripts": {
```

Comment:
I am not used to the psalm commands.. have to google them all the time.. added them here to ease contributing for non-psalm experienced people like me

Reply:
Not very happy to have `composer` as a wrapper to run scripts, but it's a non-blocker for this patch